float64 y
float64 theta
float64 ydot
float64 thetadot
